Common/Extensions/UserDefaults+LoopIntents.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UserDefaults+LoopIntents.swift
+//  Loop Intent Extension
+//
+//  Created by Anna Quinlan on 10/17/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+    
+    private enum Key: String {
+        case IntentExtensionContext = "com.loopkit.Loop.IntentExtensionContext"
+        // This key needs to be EXACTLY the same string as the objc dynamic var for the KVO to work correctly
+        case IntentExtensionOverrideToSet = "intentExtensionOverrideToSet"
+    }
+    
+    // Information for the extension from Loop
+    var intentExtensionInfo: IntentExtensionInfo? {
```

Comment:
I created `IntentExtensionInfo` so it would be easier to pass data between the intent extension and the main app in the event someone wanted to add other things to Loop's Siri capabilities